### PR TITLE
[Owner Review] Added missing enum for Digital pattern

### PR DESCRIPTION
**Pull request by ShantanuShinde**
### What does this Pull Request accomplish?

Added metadata for missing enums and generated them.
The metadata was manually added using the header file as reference

### Why should this Pull Request be merged?

The enum present in the header but missing in enums.py added

### What testing has been done?

The build runs successfully and the enums generated

### CODE REVIEW

File: source/codegen/metadata/nidigital/attributes.py [hunk 1]
```diff
@@ -899,260 +421,148 @@
...
         'type': 'ViBoolean'
     },
     1150077: {
         'access': 'read-write',
         'channel_based': False,
-        'documentation': {
-            'description': 'Specifies the maximum number of History RAM samples to acquire per site. If the property is set to -1, it will acquire until the History RAM buffer is full.\n'
-        },
         'name': 'HISTORY_RAM_MAX_SAMPLES_TO_ACQUIRE_PER_SITE',
+        'enum': 'HistoryRamMaxCyclesToAcquirePerSite',
```
Comment (doshirohan) [inline source/codegen/metadata/nidigital/attributes.py]: Cycles => Samples

**Review by doshirohan — APPROVED** [review]

File: source/codegen/metadata/nidigital/attributes.py (hunk 1 above)
Comment (ShantanuShinde) [reply source/codegen/metadata/nidigital/attributes.py]: fixed

File: source/codegen/metadata/nidigital/attributes.py [hunk 2]
```diff
@@ -763,134 +364,55 @@
...
-            'description': 'Specifies the terminal name for the output trigger signal of the specified instance of a Pattern Opcode Event. You can use this terminal name as an input signal source for another trigger.\n'
-        },
         'name': 'PATTERN_OPCODE_EVENT_TERMINAL_NAME',
         'resettable': False,
         'type': 'ViString'
     },
     1150043: {
         'access': 'read-write',
         'channel_based': False,
-        'enum': 'HistoryRamTriggerType',
```
Comment (reckenro) [inline source/codegen/metadata/nidigital/attributes.py:843]: Did you mean to remove this `'enum'` from this attribute? I think that's why it was removed in the generated proto file.

**Review by reckenro — CHANGES_REQUESTED** [review]
Left one comment about a removal of an enum. Other than that marking Request Changes similar to your other PR to see it again after it's targeted to the feature branch.

**Review by jonlugoatx — APPROVED** [review]

File: source/codegen/metadata/nidigital/attributes.py (hunk 2 above)
Comment (jonlugoatx) [reply source/codegen/metadata/nidigital/attributes.py:843]: Good catch!

Comment (ShantanuShinde) [reply source/codegen/metadata/nidigital/attributes.py:843]: fixed this

**Review by doshirohan — CHANGES_REQUESTED** [review]
Reset me when you merge from latest feature branch.

**Review by doshirohan — APPROVED** [review]

**Review by reckenro — APPROVED** [review]

**Review by jonlugoatx — APPROVED** [review]

**Pull request merged**